ui.py

```
@@ -0,0 +1,58 @@
+    def generate_expressions(self):
+        num_cnt = int(self.num_cnt_entry.get())
+        level = int(self.level_entry.get())
+        # response = requests.get(f"http://localhost:5000/expressions?num_cnt={num_cnt}&level={level}")
+        # if response.status_code == 200:
+        #     self.answer = response.json()["expression"]
+        #     nums = re.findall(r'\d+', self.answer.split('=')[0])
+        #     test=f",\t".join(nums)        
+        #     self.result_text.insert(tk.END, f"{test}\n")
+        # else:
+        #     messagebox.showerror("Error", "Failed to generate expressions")
+        self.answer=generate(num_cnt,level)
```

Ruff (0.11.9)

`generate` may be undefined, or defined from star imports

(F405)